spec/features/activity_pages/activity_pages_spec.rb

```diff
@@ -75,6 +99,49 @@
           end
         end
       end
+
+      describe 'filters' do
+        it 'has user filter' do
+          expect(page).to have_selector('#user', count: 1)
+        end
+
+        it 'user filter works' do
```

Comment:
A more descriptive name would be better, take a look at some of the other ones in this file

Reply:
Changed the names to make more sense as a sentence based on their describe block.